linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/bash/fedora.sh

```diff
@@ -0,0 +1,4 @@
+# platform = multi_platform_fedora
```

Comment:
I think that if it's `fedora.sh` then it's not required to specify this using a comment. The same for `rhel7.sh`. Am I correct?